docs(event hooks): update the doc for the lua codes (#2356)
Kong/developer.konghq.com: app/_how-tos/create-a-lambda-event-hook-with-kong-gateway.md

  config:
    functions:
      - |-
        return function (data, event, source, pid) local user = data.entity.username error("Event Hook on consumer " .. user .. "") return true end
formats:
  - admin-api
{% endentity_example %}

> ## Failure handling
> 
> We should return true if the Lua function executed succeeded. The backend engine will check the return value, if it is not true, it would be considered
> as a failure, and the backend engine will try it later.

Reviewer: ```suggestion
{:.info}
> **Note:** This should return true if the Lua function executed succeeded. The backend engine will check the return value, if it is not true, it would be considered as a failure, and the backend engine will try it later.
```
